build.sbt

```diff
@@ -1,6 +1,8 @@
 import sbtslamdata.BuildInfo
 import slamdata.Publish
 
+resolvers in ThisBuild += Resolver.bintrayIvyRepo("djspiewak", "ivy")
```

Comment:
Can we create a slamdata account so we don't have to use your personal one? In general I'd prefer our using company accounts rather than personal ones.

Reply:
We have a slamdata account.  This was mostly done as a super-emergency hack (I actually used `set` in an SBT shell opened up to sbt-bintray to redirect the publication), and it was vastly, vastly easier to get it publishing to my bintray than our company one.

Reply:
We need to get publishing, and fast. So hack understood.

I'd like us to switch this over to the company account when we have the resources to do so.